src/proto/Stellar.proto

```diff
@@ -20,34 +52,36 @@ message MemoHash {
 
 // Input data necessary to create a signed transaction.
 message SigningInput {
-    int64 amount = 1;
+    int32 fee = 1;
 
-    int32 fee = 2;
+    int64 sequence = 2;
 
-    int64 sequence = 3;
+    // Optional time bound from
+    int64 valid_after = 3;
```

Comment:
is `valid_after` and `valid_before ` necessary? can we remove them?

Reply:
Moved to ChangeTrust op only.